telemetry/telemetrybuffer.go

```diff
@@ -323,6 +323,17 @@ func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, teleme
 	}
 }
 
+// Connect to Telemetry Service without spawning new process
+func (tb *TelemetryBuffer) ConnectToTelemetry() error {
+	if err := tb.Connect(); err != nil {
```

Comment:
The parameters are used to retry the connection after starting a new one and since we are not starting the service, i removed that part.
```
StartTelemetryService(path, args)
WaitForTelemetrySocket(telemetryNumRetries, time.Duration(telemetryWaitTimeInMilliseconds))
```